app/views/user_mailer/account_reset_granted.html.erb

```diff
@@ -17,8 +17,6 @@
             </tbody>
           </table>
         </td>
-        <td class="half">
```

Comment:
Not sure we still need this anymore. I think with the way the page looks without is better, but if we still want it to be half as large we can increase it.

Reply:
Yeah, I like it full-width, and I think it's more consistent that way 👍 

It looks like this was the only usage of the `half` class. Can we remove it now?

https://github.com/18F/identity-idp/blob/66b1b4e6a5ba19ba6c3fecca3358daab72c41445/app/assets/stylesheets/email.css.scss#L78-L80